source/features/repo-age.tsx

```diff
@@ -30,21 +30,22 @@ const getFirstCommitDate = cache.function(async (): Promise<string | undefined>
 	}
 
 	if (commitsCount === 1) {
-		return select('.commit-tease-sha + span relative-time')!.attributes.datetime.value;
+		return [select('.commit-tease-sha + span relative-time')!.attributes.datetime.value, commitUrl];
 	}
 
-	const relativeTime = await fetchDom(
+	const commit = await fetchDom(
 		`${getRepoURL()}/commits?after=${commitSha}+${commitsCount - 2}`,
-		'.commit-meta relative-time'
+		'.commit-group .commit'
 	);
-
-	return relativeTime!.attributes.datetime.value;
+	const timeStamp = select('.commit-meta relative-time', commit)!.attributes.datetime.value;
+	const href = select<HTMLAnchorElement>('a.message', commit)!.href;
+	return [timeStamp, href];
 }, {
 	cacheKey: () => __filebasename + ':' + getRepoURL()
```

Comment:
This solves the "old cache" issue
```suggestion
	cacheKey: () => __filebasename + ':' + getRepoURL(),
	isExpired: value => typeof value === 'string'
```